opal/mca/btl/tcp/btl_tcp.c

```diff
@@ -60,6 +60,37 @@ mca_btl_tcp_module_t mca_btl_tcp_module =
          },
      .tcp_endpoints_mutex = OPAL_MUTEX_STATIC_INIT};
 
+void mca_btl_tcp_put_response(mca_btl_base_module_t *btl,
+                              const mca_btl_base_receive_descriptor_t *desc)  {
+
+    mca_btl_tcp_frag_t *frag = NULL;
+
+    MCA_BTL_TCP_FRAG_ALLOC_USER(frag);
+    if (OPAL_UNLIKELY(NULL == frag)) {
+        assert(0);
```

Comment:
You could just as well write `assert(NULL != frag)` which would at least provide some hint in the assert message on what went wrong ;)

Reply:
Thanks - changed it to match the others.